src/mjlab/scripts/train.py

```diff
@@ -141,6 +142,34 @@ def run_train(task_id: str, cfg: TrainConfig) -> None:
   env.close()
 
 
+def launch_training(task_id: str, args: TrainConfig | None = None):
+  args = args or TrainConfig.from_task(task_id)
+
+  import torchrunx
+
+  # torchrunx redirects stdout to logging.
+  logging.basicConfig(level=logging.INFO)
+
+  env_vars = {"MUJOCO_GL": "egl"}
+
+  if args.gpus == "all":
+    import torch.cuda
+
+    num_gpus = torch.cuda.device_count()
+  else:
+    num_gpus = len(args.gpus)
+    env_vars["CUDA_VISIBLE_DEVICES"] = ",".join(map(str, args.gpus))
```

Comment:
One concern I have here is that folks might be surprised to find the `CUDA_VISIBLE_DEVICES` variable in their env ignored:
```
export CUDA_VISIBLE_DEVICES=0
uv run train [...]
```
This is one reason why I had `--num-gpus` originally, it seems more compatible with "standard" PyTorch workflows?

Reply:
Agreed as well, CUDA_VISIBLE_DEVICES has always been my safety net for running things on multi-GPU machines and I'd expect it to behave the same in mjlab.

Reply:
At your discretion, you could also consider:

(1) Also omitting `--num-gpus` and making `CUDA_VISIBLE_DEVICES` the only control.

(2) Use `gpus: list[int] | "all"` to pass indices corresponding to `CUDA_VISIBLE_DEVICES`. I think it's slightly more flexible than `--num-gpus` which force indices `[0, ..., num_gpus - 1]`.

Reply:
I also prefer 2 and went ahead and implemented it. Thanks @apoorvkh!

Reply:
I just tried training on CPU and it's ridiculously slow and the kernels keep reloading (which points to a potential warp issue). So I upgrade my "not encouraged" to "strongly discouraged".

Reply:
Done!